casepro/backend/junebug.py

```diff
@@ -40,6 +46,73 @@ def get_addresses(self, uuid):
         return (
             a['address'] for a in addresses if a.get('address') is not None)
 
+    def get_identities(self, **params):
+        '''Get the list of identities filtered by the given kwargs.'''
+        url = '%s/api/v1/identities/search/?' % self.base_url
+
+        identities = self.get_paginated_response(
+            url, params=params)
+
+        # Users who opt to be forgotten from the system have their details
+        # stored as 'removed'
+        return (
+            IdentityStoreContact(i) for i in identities if
+            i.get('details').get('name') is not "removed"
+        )
+
+
+class IdentityStoreContact(object):
+    """
+    Holds identity data for syncing
+    """
+    def __init__(self, json_data):
+        for k, v in json_data.items():
+            setattr(self, k, v)
+
+        # Languages in the idenity store have the country code at the end
+        remote_language = json_data.get('details').get('preferred_langage')
```

Comment:
Is the spelling of `preferred_langage` correct? It seems to be missing a `u` here and in tests?